### PR TITLE
Add .pre-commit-hooks.yaml update step to RELEASE.md

### PR DESCRIPTION
**Pull request by mathbunnyru**
### What I did
Updated the docs to reflect the necessity of updating a config file.

### CODE REVIEW

**Comment by mathbunnyru** [comment]
It seems that linter has updated automatically and this triggered an error.

File: docs/RELEASE.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Only `master` branch is used for releases.
 
+1.  Update _hadolint-docker_ entry version in the
+    [.pre-commit-hooks.yaml](../.pre-commit-hooks.yaml) file.
+    Use the new version you're going to release.
+    And push to master.
+
```
Comment (m-ildefons) [inline docs/RELEASE.md:9]: This line causes the warning in remark-lint. There may only be single empty lines between list items.

**Review by m-ildefons — CHANGES_REQUESTED** [review]
LGTM otherwise.

File: docs/RELEASE.md (hunk 1 above)
Comment (mathbunnyru) [reply docs/RELEASE.md:9]: Thanks, I didn't notice it.

**Review by m-ildefons — APPROVED** [review]

**Pull request merged**